packages/cinterop/src/commonMain/kotlin/io/realm/interop/RealmInterop.kt

```diff
@@ -24,7 +24,9 @@ inline class ColumnKey(val key: Long)
 @Suppress("FunctionNaming", "LongParameterList")
 expect object RealmInterop {
 
+    fun realm_get_version_id(realm: NativePointer): Long
```

Comment:
I guess we could now go for an inline class. They even seem to be stable in 1.5, but I guess we could just address it when updating. There are also other key types that could be converted to improve type safety.